modules/neovim/init/basic.nix

```diff
@@ -158,112 +160,138 @@ in {
       default = "sensitive";
       description = "Set the case sensitivity of search";
     };
+
+    undoFile = {
+      enable = mkEnableOption "undofile for persistent undo";
+      path = mkOption {
+        type = either str luaInline;
+        default = mkLuaInline "vim.fn.stdpath('state') .. '/nvf/undo'";
+        defaultText = literalMD ''
+          ```nix
+          mkLuaInline "vim.fn.stdpath('state') .. '/nvf/undo'"
```

Comment:
Remove `/nvf` from the path, that is already included by stdpath. For example that function call returns `/home/diniamo/.local/state/nvf` for me.